src/utilities/findUserWithFullName/findUserWithFullName.test.js

```diff
@@ -0,0 +1,185 @@
+import { findUserWithFullName } from './findUserWithFullName';
+
+describe('findUserWithFullName()', () => {
+  const usersArray = [
```

Comment:
This is not the correct way of sharing a variable across tests. Please take a look at this section again
![image](https://github.com/user-attachments/assets/e8b8c994-0aee-4616-87d1-d3087ba74489)

https://doc.clickup.com/24538860/d/h/qcvqc-2728/32828824dace70e